lib/scheduler/base_config.go

```diff
@@ -65,39 +60,83 @@ func (bc BaseConfig) Validate() (errors []error) {
 	if bc.Name == "" {
 		errors = append(errors, fmt.Errorf("scheduler name shouldn't be empty"))
 	}
-	if bc.Type == "" {
-		errors = append(errors, fmt.Errorf("missing or empty type field"))
-	}
-	if bc.Percentage < minPercentage || bc.Percentage > 100 {
+	if !schedulerNameWhitelist.MatchString(bc.Name) {
 		errors = append(errors, fmt.Errorf(
-			"percentage should be between %f and 100, but is %f", minPercentage, bc.Percentage,
+			"the scheduler name should contain only numbers, latin letters, underscores, and dashes",
```

Comment:
Can we have this as a constant right next to the `schedulerNameWhitelist` ?

Reply:
done